### PR TITLE
Log formatter flag

### PR DESCRIPTION
**Pull request by eimarfandino**
a flag for forcing colored output

### CODE REVIEW

**Comment by otaviof** [comment]
@eimarfandino what if we invert this behavior? as in having the colors as a default behavior, and adding a flag to output plain text log?

I think that would be more straight forward, and also comparing to other command-line tools that actually the type of options we found.

**Comment by eimarfandino** [comment]
sounds good. I'll make the changes ready and let you know @otaviof

**Comment by eimarfandino** [comment]
@otaviof changed it to be `raw` instead of `force-colors`.

File: pkg/galaxy/landscaper.go [hunk 1]
```diff
@@ -155,7 +155,14 @@ func (l *Landscaper) loadKubeClient() error {
 // NewLandscaper instance a new Landscaper object.
 func NewLandscaper(
 	cfg *LandscaperConfig, kubeCfg *KubernetesConfig, env *Environment, ctxs []*Context,
-) *Landscaper {
```
Comment (otaviof) [inline pkg/galaxy/landscaper.go]: @eimarfandino formatting here seems a bit off, would you check?

**Comment by otaviof** [comment]
@eimarfandino aside of a minor comment, it seems really good, bro! Thanks.

**Comment by eimarfandino** [comment]
@otaviof where is the comment?

File: pkg/galaxy/landscaper.go (hunk 1 above)
Comment (eimarfandino) [reply pkg/galaxy/landscaper.go]: @otaviof i have pushed it

**Pull request merged**